### DISCUSSION TITLE
I downloaded your templates first without finding out what you're expert in. can give me a primer?

### DISCUSSION THREAD

**Discussion by simkimsia** [BryanJenksCommunity/FAQ#114] [Q&A] [answered]
hi there, this is a bit funny.

I found your link to download your templates for obsidian (i use that) and mermaid (i sometimes use that) from a youtube you did on ADHD. 

But I'm a bit lost about using your templates. Is there a webpage that explains how to use your templates in greater detail?

**Comment by tallguyjenks** [comment, marked as answer]
Ive made a video about mermaid and a video specifically about templates on my channel, a dedicated updated video on some of these more popular topics is coming soon as well:

https://www.youtube.com/c/BryanJenksTech?sub_confirmation=1
